src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$blue-color: #00acdc;
+$white-color: #fff;
+$grey-color: #616070;
+$black-color: #060b35;
+$card-width: 200px;
+$card-padding: 32px 0 16px;
+$column-gap: 48px;
+$row-gap: 46px;
+$max-container-width: $card-width * 4 + 232;
```

Comment:
What do 4 and 232 mean? Don't use mystery numbers. Create variables for them as well